src/Pos/Client/Txp/Util.hs

```diff
@@ -126,25 +157,27 @@ overrideTxOutDistrBoot c oldDistr = do
       then pure oldDistr
       else do
           when (unsafeGetCoin c < fromIntegral (length genStakeholders)) $
-               throwError $
+               throwTxError $
                sformat ("Can't spend "%build%" coins: amount is too small for boot "%
                         " era and can't be distributed among genStakeholders") c
           -- TODO CSL-1351 boot stakeholders' weights are not used
           pure $ genesisSplitBoot (HM.fromList $ map (,1) genStakeholders) c
 
 -- | Same as 'overrideTxOutDistrBoot' but changes 'TxOutputs' all at once
-overrideTxDistrBoot ::
-       (TxCreateMode ctx m) => TxOutputs -> ExceptT Text m TxOutputs
+overrideTxDistrBoot
```

Comment:
I think ideally `overrideTxDistrBoot` functions shouldn't be exported, because for me the look like an internal detail of other tx creation functions. All tx creation functions should use it. And it's quite useless separately.